sos/report/plugins/slurm.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin, RedHatPlugin
+
+
+class Slurm(Plugin, UbuntuPlugin, RedHatPlugin):
+
+    plugin_name = 'slurm'
+    profiles = ('hpc',)
+    packages = (
+        # Ubuntu
+        'slurm-wlm',
+        'slurmd',
+        'slurmdbd',
+        'slurmctld',
+        # EL
+        'slurm',
+        'slurm-slurmctld',
+        'slurm-slurmd',
+        'slurm-slurmdbd',
+    )
+    services = (
+        'slurmd',
+        'slurmdbd',
+        'slurmctld',
+    )
+
+    def setup(self):
+        """ Slurm Workload Manager
+        """
+
+        self.add_copy_spec('/etc/slurm/')
```

Comment:
Can't be there some credentials? Reading https://slurm.schedmd.com/slurm.conf.html, `AccountingStoragePass` or `JobCompPass` seems storing passwords..?